Zend/zend_compile.c

```diff
@@ -6680,6 +6675,21 @@ void zend_compile_class_const_decl(zend_ast *ast, zend_ast *attr_ast) /* {{{ */
 }
 /* }}} */
 
+void zend_compile_class_const_group(zend_ast *list) /* {{{ */
```

Comment:
```suggestion
void zend_compile_class_const_group(zend_ast *ast) /* {{{ */
```

Calling this `list` is unusual when it's not a list AST node.

Reply:
I was matching `zend_compile_prop_group()` signature for consistency. Anyways, I have renamed both instances from `list` to `ast`, thanks!